src/main/resources/static/js/welcome.js

```diff
@@ -39,17 +37,8 @@ console.log("user = " + sessionStorage.getItem("user"));
 
 /* functions */
 function newUser() {
```

Comment:
the newUser function is a bit strange in the sense that it seems to always give the user a name, even if the prompt is cancelled. I think this function should “force” the user to give a valid username, else it should be stuck in the prompt